### DISCUSSION TITLE
Welcome to flutter_map Discussions!

### DISCUSSION THREAD

**Discussion by johnpryan** [fleaflet/flutter_map#789] [General]
## flutter_map Discussions
Discussions are a new GitHub feature for connecting with other users. Comment below with an introduction of yourself and share how you're using flutter_map!

**Comment by ibrierley** [comment]
I don't mind starting the ball rolling :). 

Heya John et al, firstly thanks for all the great work on the project from everyone :). 

I'm Ian in the UK, and normally a Perl programmer :), and also a bit of history with stuff like SVG. I like trail running and walking (did a little leaflet.js basic walking routes near me site at https://mappy.dabbles.info/ )

I've done a couple of plugins at line editor https://github.com/ibrierley/flutter_map_line_editor and dragmarkers at https://github.com/ibrierley/flutter_map_dragmarker (reminder, must add them to the readme!).

The main thing I wanted to do originally was a routes/routing map app for runners (and walkers, cyclists etc) on an Android Wear watch. Integrate with directions, draw routes to a trail, show the speed and where you are walking, and be able to save the routes as gpx etc. Maybe link with buddies who are walking. I pretty much did it all, but then felt it was a bit messy code wise, so wanted to redo it now I understand it a bit better, and write better code (maybe with it all as plugins).

I tend to keep getting sidetracked though, as I'm always interested in progressing things not available (I was looking at vector tile maps for example, but I can't quite get the performance I want yet on pinch/zoom if anyone is ever interested in collaborating)

Maybe this ended up a bit much! But Hello All again! :)

**Comment by moovida** [comment]
Hi there everyone, I am Andrea and am in the field of open source GIS many years now. After maintaining for over 10 years the Android digital field mapping app geopaparazzi ( https://www.geopaparazzi.org/geopaparazzi/index.html ). I started the app from scratch using flutter and the experience gained through geopaparazzi. The result is SMASH ( https://www.geopaparazzi.org/smash/index.html ). The obvious choice for this has been flutter_map, so here I am. 
Needless to say that I am happy to see this "discussions" channel get born. I hope this can at least fill a bit the hole left by a missing mailinglist. :-)

**Comment by ibrierley** [comment]
Anyone also feel free to say what you're up to, even if you have nothing working, or even failed with it. It's often interesting and useful to hear what people are trying, what they are struggling with etc.

**Comment by ArshdeepSahni** [comment]
I wanted one **Map API of CrowdSourcing** which fits and aligns well with my **flutter App**.
Please Discuss it further if anyone has any `resource` please share here.

**Reply by ibrierley** [reply]
Heya, could you expand a bit on what you mean with Map API of Crowdsourcing ? Not sure I fully understand.
